linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny_root/oval/shared.xml

```diff
@@ -12,15 +12,14 @@
       <description>The root account should be configured to deny access after the number of defined
       failed attempts has been reached.</description>
     </metadata>
-    <criteria>
-      <criterion test_ref="test_pam_faillock_preauth_silent_system-auth"
-      comment="pam_faillock.so preauth silent set in system-auth" />
-      <criterion test_ref="test_pam_faillock_authfail_deny_root_system-auth"
-      comment="pam_faillock.so authfail deny_root value set in system-auth" />
-      <criterion test_ref="test_pam_faillock_preauth_silent_password-auth"
-      comment="pam_faillock.so preauth silent set in password-auth" />
-      <criterion test_ref="test_pam_faillock_authfail_deny_root_password-auth"
-      comment="pam_faillock.so authfail deny_root value set in password-auth" />
+    <criteria operator="OR">
+      <extend_definition comment="PAM not installed" definition_ref="package_pam_removed" />
+      <criteria>
+        <criterion test_ref="test_pam_faillock_preauth_silent_system-auth" comment="pam_faillock.so preauth silent set in system-auth" />
+        <criterion test_ref="test_pam_faillock_authfail_deny_root_system-auth" comment="pam_faillock.so authfail deny_root value set in system-auth" />
+        <criterion test_ref="test_pam_faillock_preauth_silent_password-auth" comment="pam_faillock.so preauth silent set in password-auth" />
+        <criterion test_ref="test_pam_faillock_authfail_deny_root_password-auth" comment="pam_faillock.so authfail deny_root value set in password-auth" />
```

Comment:
In updating this OVAL, it looks like there is a bug in existing content. Current OVAL requires changes to all files listed... shouldn't they be ``criteria``'d by system-auth vs password-auth? Or are both really needed?

Reply:
Should have checks even rules for both `system-auth` and `password-auth`

Reply:
Of course, that's until we can get a software dev to create faillock.conf so that we don't have to do this finagling anymore.